src/.pre-commit-config.yaml.jinja

```diff
@@ -31,7 +31,7 @@
   {%- set repo_rev.prettier = "2.4.1" %}
   {%- set repo_rev.prettier_xml = "1.1.0" %}
   {%- set repo_rev.pylint = "v2.11.1" %}
-  {%- set repo_rev.pylint_odoo = "5.0.5" %}
+  {%- set repo_rev.pylint_odoo = "7.0.0" %}
```

Comment:
@moylop260 any check has changed in its default behavior during these version?

Reply:
The summary of the [Changelog](https://pypi.org/project/pylint-odoo#CHANGES) is:

```txt
7.0.0
[REF] external-request-timeout: Add ftplib.FTP method (#388)
[FIX] missing-import-error: add xlrd to whitelist (#379)

6.2.0
[IMP] duplicate-po-message-definition: Detecting duplicated only with msgid (#374)
[IMP] test-folder-imported: Consider ‘from . import test’ missing case (#373)

6.1.0
[ADD] external-request-timeout: It could wait for a long time (#370)

6.0.0
[ADD] pre-commit-hooks: Add pylint_odoo hook to be used from pre-commit.yaml directly (#368)
[REF] requirements: Bump pylint package from 2.11.1 to 2.13.5 for py3.x (#367)
[IMP] sql-injection: Detect possible sql injections when using f-strings (#364)
[FIX] Pinned lazy-object-proxy for Python 2.7 (#365)
[FIX] missing-return: Incorrect check for tearDownClass (#362)
```

Notice that f-strings are considered as sql-injection too and other changes for other checks

As @yajo said It shouldn't raise errors but could

If a project is using f-string in a sql-injection way it will be red

Check the other checks changes and check if the configuration files have enabled and so on

But even a sql-injection is better fixed even late than early

Reply:
Yes, I think it's good to improve such check. Thanks for the info.